packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx

```diff
@@ -149,6 +157,25 @@ export const WalletManagementTile = () => {
   const { isEnabled: isStripeEnabled } = useFlag(
     FeatureFlags.BUY_AUDIO_STRIPE_ENABLED
   )
+  const coinbaseAllowedCountries = (
+    useRemoteVar(StringKeys.COINBASE_PAY_ALLOWED_COUNTRIES) ?? ''
+  ).split(',')
+  const stripeAllowedCountries = (
+    useRemoteVar(StringKeys.STRIPE_ALLOWED_COUNTRIES) ?? ''
+  ).split(',')
+  const stripeDeniedRegions = (
+    useRemoteVar(StringKeys.STRIPE_DENIED_REGIONS) ?? ''
+  ).split(',')
+
+  const { value } = useAsync(getLocation, [])
+
+  const isCoinbaseAllowed =
```

Comment:
should we memo these? we're iterating over a decently large array with ever re-render otherwise